src/main/java/org/isf/accounting/manager/BillBrowserManager.java

```diff
@@ -178,37 +185,40 @@ private Bill newBill(Bill newBill) throws OHServiceException {
 
 	/**
 	 * Stores a list of {@link BillItems} associated to a {@link Bill}.
+	 * 
 	 * @param billID the bill id.
 	 * @param billItems the bill items to store.
 	 * @throws OHServiceException
 	 */
 	private void newBillItems(int billID, List<BillItems> billItems) throws OHServiceException {
 		ioOperations.newBillItems(ioOperations.getBill(billID), billItems);
 	}
-	
+
 	/**
 	 * Stores a list of {@link BillPayments} associated to a {@link Bill}.
+	 * 
 	 * @param billID the bill id.
 	 * @param payItems the bill payments.
 	 * @throws OHServiceException
 	 */
 	private void newBillPayments(int billID, List<BillPayments> payItems) throws OHServiceException {
 		ioOperations.newBillPayments(ioOperations.getBill(billID), payItems);
 	}
-	
+
 	/**
 	 * Updates the specified {@link Bill} along with all its {@link BillItems} and {@link BillPayments}
+	 * 
 	 * @param updateBill - the bill to update.
 	 * @param billItems - the list of bill's items
 	 * @param billPayments - the list of bill's payments
```

Comment:
```suggestion
	 * @param billPayments the list of bill's payments
```
And there are about 60+ more like this

Reply:
anyway this is the way I see the javadoc, doesn't look so bad...

![image](https://github.com/user-attachments/assets/e7404c97-9e76-4cd7-86df-bc0faca6ae44)

Reply:
anyway, I took the chance to revise some of them